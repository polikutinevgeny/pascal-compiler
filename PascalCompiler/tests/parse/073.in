var
   a: integer;
   function foo(): integer;
   begin
      result := 4;
   end;
begin
   write(a, foo());
end.