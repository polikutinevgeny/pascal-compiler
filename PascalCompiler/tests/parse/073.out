Illegal identifier at 5:7
